halfdiamond.c

```diff
@@ -0,0 +1,27 @@
+/*Rumeet Goradia - Half Diamond Program*/
```

Comment:
Perfect job! 💯 I love that you prompted the user for the length rather than just picking an arbitrary number. Awesome work! 👍